src/frequenz/sdk/timeseries/logical_meter/_logical_meter.py

```diff
@@ -60,7 +60,7 @@ async def _engine_from_formula_string(
             self._resampler_subscription_sender,
             metric_id,
         )
-        return await builder.from_string(formula)
+        return await builder.from_string(formula, False)
```

Comment:
Reading this, I recommend making the `nones_are_zeros` arguments a keyword-only argument (`def f(..., *, nones_are_zeros: bool`), otherwise is very hard to know what's going on on the caller site without having to go see the function definition.

Reply:
https://github.com/frequenz-floss/frequenz-sdk-python/issues/100